### PR TITLE
Fix cql 0.7.0 build version

### PR DESCRIPTION
**Pull request by auxten**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by auxten** [comment]
Squash Merge, pls

**Comment by jonchang** [comment]
Does this need to have its `revision` bumped?

File: Formula/cql.rb [hunk 1]
```diff
@@ -16,10 +16,10 @@ class Cql < Formula
 
   def install
     ENV["GOPATH"] = buildpath
-    ENV["CQLVERSION"] = "v0.6.0"
+    ENV["CQLVERSION"] = "v0.7.0"
```
Comment (SMillerDev) [inline Formula/cql.rb]: please use `ENV["CQLVERSION"] = "v#{version}"`

File: Formula/cql.rb [hunk 2]
```diff
@@ -16,10 +16,10 @@ class Cql < Formula
...
     ENV["CGO_ENABLED"] = "1"
     mkdir_p "src/github.com/CovenantSQL"
-    ldflags = "-X main.version=v0.6.0 " \
+    ldflags = "-X main.version=v0.7.0 " \
```
Comment (SMillerDev) [inline Formula/cql.rb]: same here

**Review by SMillerDev — CHANGES_REQUESTED** [review]

**Comment by auxten** [comment]
@jonchang yes, #41274

**Comment by SMillerDev** [comment]
Please add a revision bump then.

**Comment by fxcoudert** [comment]
> Please add a revision bump then

Please add a revision bump and squash everything into a single commit, with message `cql: fix version`

**Comment by auxten** [comment]
Squash done.
@SMillerDev @fxcoudert version has been bumped by some other guy in #41274

**Comment by SMillerDev** [comment]
>Squash done. ...

@auxten I know. Your PR needs a line that says `revision 1` though, or the change will never get to the users.

**Comment by auxten** [comment]
@SMillerDev Oh, I see. done

**Review by SMillerDev — APPROVED** [review]

**Comment by SMillerDev** [comment]
Please fix the audit issue.

**Comment by auxten** [comment]
@SMillerDev fixed, sorry for so much trouble.

**Comment by SMillerDev** [comment]
Not a problem at all, I'd much rather have this then someone who doesn't do anything to improve homebrew for all it's users

**Comment by fxcoudert** [comment]
Thanks @auxten

**Pull request closed without merging**